### ISSUE TITLE
Currupted about-data.c

### ISSUE THREAD

**Issue by huseyinkozan** [udda/multiload-ng#3]
Hello,
While trying to compile, I got this file and breaks compilation:
```c
#include <glib.h>

const gchar * const about_data_authors[] = {
	"Mario Cia",
	"ciolo <mr.udda@gmail.com>",
	"a",
	"dhp <",
	"a",
	"dhp@gmail.com>Marti",
	" Baulig <marti",
	"@home-of-li",
	"ux.org>Todd Kulesza <fflewddur@dropli",
	"e.",
	"et>Be",
	"oît Dejea",
	" <TazForEver@dlfp.org>Davyd Madeley <davyd@madeley.id.au>",
	NULL
};

const gchar *about_data_icon = "utilities-system-monitor";

```

**Comment by udda** [comment]
Hello, thank you very much for the report.

I think that the problem is that your *sh* does not properly parse custom field separators (**$IFS** shell variable). I am using Gentoo and here *sh* is a symlink to *bash*, so this is not a problem to me, but \*ubuntu users do have a separate *sh* program (*dash*).

The file **about-data.c** is dynamically generated by **generate-about-data.sh**, a simple shell script in the project root. Your **about-data.c** is using `n` as line separator instead of `\n`. I think that simply changing the shebang to point to *bash* would fix the problem.

I'm not worried about compatibility issues as virtually every distro now has bash installed by default. 



**TL;DR**: I just pushed a commit that should fix your issue. Please let me know.

**Comment by huseyinkozan** [comment]
Thank you for the app and fix : ^)
PS: I am using zsh, if matters.
